Updates Ionyweb to use z-index greater than 9000

chrisfranklin opened the pull request:
Hey,

This should fix issue #14 as per your instructions, I have not had a chance to test this yet.

Cheers,

Chris

chrisfranklin committed:
Update Z index for iony admin panels so they are on top

Set all Ionyweb z-index values above 9000 so the admin
always displays above the theme if the user follows the
defined standard of not using any index above 9000.

chrisfranklin committed:
Documents z-index standard for Ionyweb admin panels

Adds documentation warning the user not to set the
value of z-index above 9000 so as not to conflict
with Ionyweb

Natim commented on ionyweb/static/admin/css/ionyweb_admin.less:
```diff
@@ -100,7 +100,7 @@
 		/* Each UL is a list of action in admin_tool_bar*/
 		ul.toolbar_menu{
-			z-index: 1100;
+			z-index: 9100;
```
900 to 9100,
950 to 9200,
1000 to 9300,
1100 to 9400

It should be 9400 here I think.

Natim commented:
There is just a little question with line 103 of ionyweb_admin.less and I think we will be good to merge :)

chrisfranklin committed:
Fix z-index from 9100 to 9400 originally converted from 1100

Corrected the wrong z-index as per these instructions:

900 to 9100,
950 to 9200,
1000 to 9300,
1100 to 9400

Natim commented on ionyweb/static/admin/css/ionyweb_admin_dark.less:
```diff
@@ -18,7 +18,7 @@
 @background_hover: rgba(0, 0, 0, 0.2); /* background of the admin section : also #444 or #999*/
 
-@zindex: 1000;
+@zindex: 9000;
```
1000 to 9300

chrisfranklin replied:
Some of the classes perform arithmetic on this value, does this matter?

Natim replied:
Maybe...

Natim replied:
I checked, it should be ok.

chrisfranklin commented:
That should be all, thanks for the nice easy one to get me started =) have you guys got a roadmap or a list of features / bugs that you would like people to work on?

Natim commented on ionyweb/static/admin/css/ionyweb_login.less:
```diff
@@ -14,7 +14,7 @@
 @background_widget: rgba(220, 220, 220, 0.3); /* background of the admin section : also #444 or #999*/
 @background_hover: rgba(220, 220, 220, 0.1); /* background of the admin section : also #444 or #999*/
-@zindex: 1000;
+@zindex: 9000;
```
1000 to 9300

Natim commented:
I am a bit perfectionist I guess.

Can you also run make less to compile them for production ?

chrisfranklin committed:
Fix 1000 being set to 9000 when it should be 9300

Fix wrong z-index and recompile less files for production

chrisfranklin commented:
Sorry for this being quite such a mess, do I need to make any other changes to get this included? I promise to be more precise in future.

almet commented:
@Natim ping

Natim commented on ionyweb/static/admin/css/$:
```diff
@@ -0,0 +1,1948 @@
+/* General administration variables */
```
Strange filename.

Natim commented on ionyweb/static/admin/css/ionyweb_login.min.css:
```diff
@@ -1,21 +1,236 @@
-@font-face{font-family:'MarketingScriptRegular';src:url('fonts/MarketingScript-webfont.eot');src:url('fonts/MarketingScript-webfont.eot?#iefix') format('embedded-opentype'),url('fonts/MarketingScript-webfont.woff') format('woff'),url('fonts/MarketingScript-webfont.ttf') format('truetype'),url('fonts/MarketingScript-webfont.svg#MarketingScriptRegular') format('svg');font-weight:normal;font-style:n ...
```
not minified anymore?

Natim commented:
I definitly want to merge that thank you and sorry for the delay.
